src/style.css

```diff
@@ -1,3 +1,75 @@
+html {
+  --accent-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+}
+
+.container {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+@media screen and (max-width: 1199px) {
+  .container {
+    width: 1024px;
+  }
+}
+
+.logo__link {
+  line-height: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
   margin: 0;
+  padding: 0;
+}
+
+.nav__list :nth-child(n+2) {
+  margin-left: 20px;
+}
+
+.nav__item {
+  position: relative;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  line-height: 60px;
```

Comment:
you can try to remove it and set height for container, then center elements inside
It can help to pass a tests and follow requirements